### DISCUSSION TITLE
DT adjustment increments with arrow keys is reduced to 0.01 after 2023.924.0

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#24918] [Q&A] [answered]
### Type

Game behaviour

### Bug description

In previous versions of lazer, the adjustment for DT with arrow keys was 0.1; 2023.924.0 reduced that to 0.01. I am making this a bug report because the change wasn't mentioned in the update notes, and having adjustment be 0.1 instead of 0.01 seems like deliberate behaviour. Presumably this was changed in https://github.com/ppy/osu/pull/24801

I think that having the ability to change by 0.01 is useful, and being forced to do so with the mouse is finnicky, but I also want the ability to change in increments of 0.1. The up/down arrow keys are unused, so I think that splitting the functionality between up/down and left/right would be the best option, although this logic could be applied to basically any slider. Might be best to build it into slider functionality as a whole, but afformentioned up/down keys might not be unused everywhere

### Screenshots or videos

current:
https://github.com/ppy/osu/assets/49025619/0ccd4b1e-cb66-40fd-ab57-55d5f17f886e

previous:
https://github.com/ppy/osu/assets/49025619/c3394859-a291-40b2-9ad9-01d845e345b0

### Version

2023.924.0

### Logs

[network.log](https://github.com/ppy/osu/files/12711163/network.log)
[performance.log](https://github.com/ppy/osu/files/12711164/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12711165/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12711166/updater.log)
[database.log](https://github.com/ppy/osu/files/12711167/database.log)
[input.log](https://github.com/ppy/osu/files/12711168/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/12711169/legacy-ipc.log)

**Comment by peppy** [comment, marked as answer]
This was an intentional change, bundled with other changes. Eventually we'll add the ability to increment in large amounts holding shift or control.
